### ISSUE TITLE
MSFraggerAdapter doesn't support "cut before amino acd"

### ISSUE THREAD

**Issue by zhuchcn** [OpenMS/OpenMS#6325]
MSFragger now supports the "cut before amino acid" function since 3.4 (https://github.com/Nesvilab/MSFragger/issues/9), which is not supported by MSFraggerAdapter yet and it will be good to have this added. The parameter for this functionality seems to be `search_enzyme_sense_1` (see [here](https://github.com/Nesvilab/MSFragger/blob/90a49d7ad77f071a35a1c09a8a86a4149487ccea/parameter_files/closed_fragger.params#L30-L40)).

**Comment by jpfeuffer** [comment]
I agree! I think we should generally update the adapter (in case there were some other new parameters).
@tillenglert which version are/were you using?
